lib/chibi/test.scm

```diff
@@ -490,23 +476,36 @@
          (set-cdr! info (cons (cons 'gen-name name) (cdr info))))
      name)))
 
-(define (test-print-name info . indent)
-  (let ((width (- (current-column-width)
-                  (or (and (pair? indent) (car indent)) 0)))
-        (name (test-get-name! info)))
-    (display name)
+(define (test-print-name info indent)
+  (let* ((width (- (current-column-width) indent))
+         (name (test-get-name! info))
+         (diff (- width 9 (string-length name))))
+    (display
+     (if (positive? diff)
+         name
+         (string-append
+          (substring name 0 (+ (string-length name) diff -1))
+          (string (integer->char #x2026)))))
     (display " ")
-    (let ((diff (- width 9 (string-length name))))
-      (cond
-       ((positive? diff)
-        (display (make-string diff #\.)))))
+    (if (positive? diff)
+        (display (make-string diff (integer->char #x2024))))
     (display " ")
     (flush-output-port)))
 
 (define (test-group-indent-width group)
   (let ((level (max 0 (+ 1 (- (test-group-ref group 'level 0)
                               (test-first-indentation))))))
-    (* 4 (min level (test-max-indentation)))))
+    (* (current-group-indent) (min level (test-max-indentation)))))
+
+(define indent-string
```

Comment:
This doesn't seem right as a global.  Or at least we should have some way of resetting it.  I think it's ok for now but can you add a note to that effect?

Reply:
Implemented